discovery/discovery.go

```diff
@@ -207,6 +213,43 @@ func (o *orchestratorPool) GetOrchestrators(ctx context.Context, numOrchestrator
 	return ods, nil
 }
 
+func getModelCaps(caps *net.Capabilities) map[string]*net.Capabilities_CapabilityConstraints_ModelConstraint {
+	if caps == nil || caps.Constraints == nil || caps.Constraints.PerCapability == nil {
+		return nil
+	}
+	liveAI, ok := caps.Constraints.PerCapability[uint32(core.Capability_LiveVideoToVideo)]
+	if !ok {
+		return nil
+	}
+	return liveAI.Models
+}
+
+func reportLiveAICapacity(info *net.OrchestratorInfo, capsReq common.CapabilityComparator, orchURL *url.URL) {
```

Comment:
Sorry to take on this again, but I still think there is something wrong with these metrics. I think we don't zero them when the `info` is nil (which is the case e.g. when the GetOrchInfo returns `OrchCapped` errror).